auth0/token.go

```diff
@@ -349,22 +317,28 @@ func (a *ClientProvider) RefreshToken() (RefreshResult, error) {
 	}
 
 	if authToken == "" || err != nil {
-		authToken, err = a.refreshCachedToken()
+		_, err = a.refreshCachedToken()
 		if err != nil {
 			return RefreshError, err
 		}
+
 		return RefreshSuccessful, nil
 	}
 
-	ok, claims, err := a.isValid(authToken)
+	ok, claims, err := a.isValid(authToken, false)
```

Comment:
No need to the second input

Reply:
I think it is good to be precise by specifying the second input.